WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -485,6 +506,15 @@ extension AztecPostViewController {
     enum EditionMode {
         case richText
         case html
+
+        mutating func toggle() {
```

Comment:
Ok, I'm gave it a spin this morning and found this crashes but I think they are expected at the moment:

 - Post with photo crashes
 - Trying to add photo crashes
 - Trying to add a link on a empty post crashes

The rest of functionality is working ok, expect for the things Aaron already noticed. I will have a look on the code now.